src/index.js

```diff
@@ -82,7 +82,7 @@ function ParseServer({
   facebookAppIds = [],
   enableAnonymousUsers = true,
   oauth = {},
-  serverURL = '',
+  serverURL = (()=>{throw 'You must provide a serverURL!'})(),
```

Comment:
Sure, I don't see why not. I might actually give this pattern a name to make it more clear.